### DISCUSSION TITLE
How can I disable type checking?

### DISCUSSION THREAD

**Discussion by bglgwyng** [swc-project/swc#1951] [Q&A] [answered]
Does swc have an environment variable or a .swcrc option to disable its type checking process in typescript compilation?
I'm trying https://github.com/pradel/create-react-app-swc

**Comment by kdy1** [comment, marked as answer]
swc does not perform type checking

**Reply by bglgwyng** [reply]
Thanks for the reply. I just submitted an incomplete question by mistake.
Then I guess the type errors I saw in my console were from tsc.
